src/aliased_struct.h

```diff
@@ -0,0 +1,73 @@
+#ifndef SRC_ALIASED_STRUCT_H_
+#define SRC_ALIASED_STRUCT_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
```

Comment:
I would put the implementations here into a separate file… https://github.com/nodejs/node/commit/be780e9443bc091854879d221db490e99ff4ff45#diff-60658f40e7ef75905d41ef98a6f6b1f7R7 ;)